index.js

```diff
@@ -11,6 +11,7 @@ module.exports = function (movieString) {
     movie = baseName;
   }
 
-  return movie;
+  movie = movie.substring(movie.indexOf('-') + 1);
```

Comment:
OK, embarrassed now !
But hyphens do occur in film names, or is it by convention not in the ripped filenames?